README.md

```diff
@@ -53,7 +57,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The value is true, and weather === "sunny"  absolutely has to equal to sunny.
```

Comment:
Not entirely sure what you mean by this; you're right that the last expression evaluates to `true`, though.